src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -345,6 +345,9 @@ akka {
       # Enables SSL support on this transport
       enable-ssl = false
 
+	  # Enables backwards compatibility with Akka.Remote clients running Helios 1.*
+	  enable-backwards-compatibility = false
```

Comment:
This setting was added as Helios 2.0 needed a special adapter to talk to Helios 1.4* - may no longer be necessary as of Helios 2.1, which this package uses. I'll have to test first though.